### PR TITLE
[Docs] fix wrong director; add link

### PR DESCRIPTION
**Pull request by davidxia**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: docs/feature_rgb_matrix.md [hunk 1]
```diff
@@ -562,7 +562,9 @@ static bool my_cool_effect2(effect_params_t* params) {
 #endif // RGB_MATRIX_CUSTOM_EFFECT_IMPLS
 ```
 
-For inspiration and examples, check out the built-in effects under `quantum/rgb_matrix_animations/`
+For inspiration and examples, check out the built-in effects under [`quantum/rgb_matrix/animations/`][rgb-matrix-animations-dir].
+
+[rgb-matrix-animations-dir]: https://github.com/qmk/qmk_firmware/tree/master/quantum/rgb_matrix
```
Comment (fauxpark) [inline docs/feature_rgb_matrix.md:567]:
```suggestion
For inspiration and examples, check out the built-in effects under `quantum/rgb_matrix/animations/`.
```
Should also be done for LED Matrix if necessary.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: docs/feature_rgb_matrix.md (hunk 1 above)
Comment (davidxia) [reply docs/feature_rgb_matrix.md:567]: Thanks, I guess you don't want to link to github.com?

File: docs/feature_rgb_matrix.md [hunk 2]
```diff
@@ -562,7 +562,9 @@ static bool my_cool_effect2(effect_params_t* params) {
 #endif // RGB_MATRIX_CUSTOM_EFFECT_IMPLS
 ```
 
-For inspiration and examples, check out the built-in effects under `quantum/rgb_matrix_animations/`
+For inspiration and examples, check out the built-in effects under `quantum/rgb_matrix/animations/`.
+
+[rgb-matrix-animations-dir]: https://github.com/qmk/qmk_firmware/tree/master/quantum/rgb_matrix
```
Comment (fauxpark) [inline docs/feature_rgb_matrix.md:567]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: docs/feature_rgb_matrix.md (hunk 2 above)
Comment (davidxia) [reply docs/feature_rgb_matrix.md]: sorry forgot the link, removed

**Review by fauxpark — APPROVED** [review]

**Pull request merged**